python/modules/IcePy/Connection.cpp

```diff
@@ -591,54 +591,159 @@ connectionThrowException(ConnectionObject* self, PyObject* /*args*/)
 }
 
 static PyMethodDef ConnectionMethods[] = {
-    {"abort", reinterpret_cast<PyCFunction>(connectionAbort), METH_NOARGS, PyDoc_STR("abort() -> None")},
-    {"close", reinterpret_cast<PyCFunction>(connectionClose), METH_NOARGS, PyDoc_STR("close() -> Ice.Future")},
+    {"abort",
+     reinterpret_cast<PyCFunction>(connectionAbort),
+     METH_NOARGS,
+     PyDoc_STR("abort() -> None\n\n"
+               "Aborts this connection.")},
+    {"close",
+     reinterpret_cast<PyCFunction>(connectionClose),
+     METH_NOARGS,
+     PyDoc_STR("close() -> Ice.Future[None]\n\n"
+               "Starts a graceful closure of this connection once all outstanding invocations have completed.\n\n"
+               "Returns\n"
+               "-------\n"
+               "Ice.Future[None]\n"
```

Comment:
The method doesn't returns always return an `Ice.Future`, it calls `wrapFuture` which can return another future type. Can we use `Awaitable[None]` like for Python code?

This is true for other methods that return futures, we always call `wrapFuture`.